script/postpublish

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env ruby
 require 'json'
 
+verbose("npm run logout")
```

Comment:
Should the definition of verbose go above this?

Reply:
fixed